### PR TITLE
 workflow: set up continuous release with pkg.pr.new

### PR DESCRIPTION
**Pull request by warmthsea**
Must be installed:

https://github.com/apps/pkg-pr-new

Refer to: 

https://x.com/youyuxi/status/1824692613217390877
https://github.com/vuejs/core/blob/main/.github/workflows/ci.yml
https://github.com/stackblitz-labs/pkg.pr.new#readme

Demo:

https://github.com/warmthsea/elp-pkg-pr-release-test
https://elp-pkg-pr-release-test.vercel.app/

Test:

https://github.com/warmthsea/element-plus/pull/2#issuecomment-2316700555

### CODE REVIEW

File: .github/workflows/pkg-pr-new.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  pull_request:
+    branches:
+      - dev
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```
Comment (btea) [inline .github/workflows/pkg-pr-new.yml:17]: It is best to keep these action versions consistent with other CIs.

**Review by btea — APPROVED** [review]
LGTM

**Comment by warmthsea** [comment]
image

**Review by tolking — COMMENTED** [review]
I have a few suggestions to make

- It is better to run it after pr-docs-build is successfully executed. (like pr-docs-deploy.yml)
- It is better to keep the relevant dependency versions consistent with other CIs.

**Pull request merged**